include/exadg/grid/enum_types.h

```diff
@@ -39,7 +39,8 @@ enum class TriangulationType
 {
   Undefined,
   Distributed,
-  FullyDistributed
+  FullyDistributed,
+  Serial
```

Comment:
Why did you introduce this here? Can you please explain this in detail?

Reply:
Because there was enough to think about to get simplex running.

Reply:
What do you mean? Is Serial mandatory for simplex? Or is this change not directly related to this PR dealing with simplex?

Reply:
> Is Serial mandatory for simplex?

No. It can be extracted into separate PR.

Reply:
has been done in #116